pkg/sources/resume.go

```diff
@@ -0,0 +1,83 @@
+package sources
+
+import (
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// RemoveRepoFromResumeInfo removes the repoURL from the resume info.
+func RemoveRepoFromResumeInfo(resumeRepos []string, repoURL string) []string {
+	index := -1
+	for i, repo := range resumeRepos {
+		if repoURL == repo {
+			index = i
```

Comment:
**Nit (optional):** I know this is simply moving code around, but we can `break` out of this loop once we found the repo. Alternatively, since the `resumeRepos` are sorted, we could do a binary search using [sort.SearchStrings](https://pkg.go.dev/sort#SearchStrings)

Reply:
Ah that's neat. I do like the `break`. But looking at `sort.SearchStrings`, it could return an index that we wouldn't want to use, because it doesn't actually contain the `repoURL`, and we'd have to check that that index did exactly equal the repo. But still very cool to know about.